src/frontend/src/lib/components/transactions/TransactionsDateGroup.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import { SLIDE_DURATION } from '$lib/constants/transition.constants';
+	import type { AllTransactionsUi } from '$lib/types/transaction';
+
+	export let date: string;
+	export let transactions: AllTransactionsUi;
+</script>
+
+{#if transactions.length > 0}
+	<div class="mb-5 flex flex-col gap-4">
+		<span class="text-lg font-medium text-tertiary first-letter:capitalize">{date}</span>
+
+		{#each transactions as transaction, index (`${transaction.id}-${index}`)}
```

Comment:
Just wondering: is there a chance that 2 transactions will have the same id?

Reply:
good one! i asked myself the same: yes, it could! but the additional index, should solve the issue, since it is unique for each one

Reply:
Alrighty, thanks for the answer